Realm/RLMAccessor.mm

```diff
@@ -179,25 +179,34 @@ static inline void RLMSetValue(__unsafe_unretained RLMObjectBase *const obj, NSU
     }
 }
 
-static inline size_t RLMAddLinkedObject(RLMObjectBase *link,
-                                        __unsafe_unretained RLMRealm *const realm,
-                                        RLMCreationOptions options) {
+static inline RLMObjectBase *RLMGetLinkedObjectForValue(__unsafe_unretained RLMRealm *const realm,
```

Comment:
This probably needs to be marked as `NS_RETURNS_RETAINED` to avoid autoreleasing everything (`RLMAddLinkedObject` avoided the autorelease by returning the row index rather than the object).